kitsune/questions/models.py

```diff
@@ -708,9 +709,8 @@ def reload_from_analytics(cls, verbose=False):
         """Update the stats from Google Analytics."""
         from kitsune.sumo import googleanalytics
 
-        counts = googleanalytics.pageviews_by_question(
-            settings.GA_START_DATE, date.today(), verbose=verbose
-        )
+        one_year_ago = datetime.now() - timedelta(days=365)
```

Comment:
This'll be a `datetime`, whereas previously the first argument to `pageviews_by_question` was a `date`, could this pose a problem?

Reply:
it will just add the time portion too but that's a good flag. I am updating this back to date for consistency.